source/unified-test-format/unified-test-format.rst

```diff
@@ -1798,6 +1857,155 @@ Use a ``listIndexes`` command to check whether the index exists. Note that it is
+  test runner MUST do nothing. If the entity exists and is of a different type,
+  the test runner MUST raise an error.
+  
+  If this option is specified and ``storeErrorsAsEntity`` is not, errors
+  MUST also be captured and appended to the array.
+
+  Documents appended to the array MUST contain the following fields:
+  
+  - ``error``: the textual description of the failure encountered.
+  - ``time``: the number of (floating-point) seconds since the Unix epoch
+    when the failure was encountered.
+
+- ``storeSuccessesAsEntity``: if specfied, the runner MUST keep track of
+  the number of sub-operations that completed successfully, and store
+  that number in the specified entity. For example, if the loop contains
+  two sub-operations, and they complete successfully, each loop execution
+  would increment the number of successes by two.
+  
+  If the entity of the specified name already exists, the test runner
+  MUST raise an error.
+
+- ``storeIterationsAsEntity``: if specfied, the runner MUST keep track of
+  the number of iterations of the loop performed, and store that number
+  in the specified entity.
+  
+  If the entity of the specified name already exists, the test runner
+  MUST raise an error.
+
+A *failure* is when the result or outcome of an operation executed by the
+test runner differs from its expected outcome. For example, an ``expectResult``
+assertion failing to match a BSON document or an ``expectError`` assertion
+failing to match an error message would be considered a failure. An *error*
+is any other type of error raised by the test runner. For example, an
+unsupported operation or inability to resolve an entity name would be
+considered an error.
+
+This specification permits the test runner to report some failures as errors
+and some errors as failures. When the test runner stores errors and
+failures as entities it MAY classify conditions as errors and failures in
+the same way as it would when used in the driver's test suite.
+This includes reporting all errors as failures or all failures as errors.
+
+If the test runner does not distinguish errors and failures in its reporting,
+it MAY report both conditions under either category, but it MUST report
+any given condition in at most one category.
+
+The same entity name MAY be used for both ``storeErrorsAsEntity``
+and ``storeFailuresAsEntity`` options. The test runner MUST support
+writing both errors and failures to the same entity. If the same entity is
+specified, the errors and failures MUST be written in chronological order
+to the extent possible, interleaving the error and failure entries
+if necessary. Note that the textual description field for both errors
+and failures is ``error``.
+
+The following termination behavior MUST be implemented by the test
+runner:
+
+- The test runner MUST provide a way to request termination of loops. This
+  request will be made by the Atlas testing workload executor in response to
```

Comment:
Can we link to the Drivers Atlas Testing spec or documentation here so that it is clear what is meant by 'Atlas testing workload executor' since this is not something we mention in the `Terms` section .

Reply:
Added.